modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -17,21 +34,20 @@ AWS Load Balancer Operator can work with AWS Outposts. Application Load Balancer
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: echoserver
+  name: <application_name>
   annotations:
-    alb.ingress.kubernetes.io/subnets: <outposts_subnet_id> # <1>
+    alb.ingress.kubernetes.io/subnets: <subnet_id> # <1>
```

Comment:
There mush be at least 2 subnets from different availability zones.

Reply:
Perhaps there are some exception cases?

```
$ oc get ingress alb-outpost-public -n ${APP_NAME} -o yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    alb.ingress.kubernetes.io/scheme: internet-facing
    alb.ingress.kubernetes.io/subnets: subnet-007223c427204d300
    alb.ingress.kubernetes.io/target-node-labels: location=outposts
    alb.ingress.kubernetes.io/target-type: instance
  creationTimestamp: "2024-02-22T06:12:22Z"
  finalizers:
  - ingress.k8s.aws/resources
  generation: 1
  labels:
    location: outpost
  name: alb-outpost-public
  namespace: myapp-outpost
  resourceVersion: "42950"
  uid: a38cd630-07d0-4e0a-bd05-cdf3c3d9ef8f
spec:
  ingressClassName: cloud
  rules:
  - http:
      paths:
      - backend:
          service:
            name: myapp-outpost-alb
            port:
              number: 80
        path: /
        pathType: Prefix
status:
  loadBalancer:
    ingress:
    - hostname: k8s-myappout-alboutpo-54dc972b7a-1655616622.us-east-1.elb.amazonaws.com
```

![Screenshot from 2024-02-22 03-16-52](https://github.com/openshift/openshift-docs/assets/3216894/7aff2b78-3937-48bb-93fa-2ecd5737ede7)
![Screenshot from 2024-02-22 03-17-26](https://github.com/openshift/openshift-docs/assets/3216894/c9cfaa89-18fc-4f1e-927c-4b9198916858)

Reply:
The way this is phrased, it could be applied to either. I can add context that the non-Outpost case needs at least two in different AZs so it's completely clear.